libraries/tests/test_views.py

```diff
@@ -157,7 +157,7 @@ def test_GET_returns_200(self):
     def test_user_name_present(self):
         r = self.client.get(self.url_compare)
         actual = self.css_select_get_text(r, 'tr#username th')
-        expected = ['', self.user.username, self.other_user.username]
+        expected = ['', self.user.username, 'Difference', self.other_user.username]
```

Comment:
This test needed to be updated once the HTML fix happened.

Reply:
I was so puzzled why that item was missing from the list! Smh at myself.